contracts/Zauction.sol

```diff
@@ -12,11 +12,15 @@ contract ZAuction is Initializable, OwnableUpgradeable {
   using ECDSA for bytes32;
 
   IERC20 public token;
+  // IERC721 public nft;
   IRegistrar public registrar;
 
   // Original zAuction contract address for backward compatibility
   address legacyZAuction;
+
   mapping(address => mapping(uint256 => bool)) public consumed;
+  mapping(uint256 => uint256) public rootDomainId;
+  mapping(uint256 => uint256) public rootDomainRoyalty;
```

Comment:
Could you please make some minor tweaks to the wording/verbiage.

Instead of "royalty" for the top-level domains, can this be referred to as "fee" ie `rootDomainFee`

Additionally, root domain may be misleading, since the actual 'root' domain is id 0x0, instead this really represents the "top" level domain (similar to .com) so can we change reference of "root domain" to "top level domain" please? ie `topLevelDomainId` and `topLevelDomainRoyalty`